fs/fs_zenfs.cc

```diff
@@ -1385,9 +1385,10 @@ IOStatus ZenFS::MigrateFileExtents(
   IOStatus s = IOStatus::OK();
   Info(logger_, "MigrateFileExtents, fname: %s, extent count: %lu",
        fname.data(), migrate_exts.size());
+
   // The file may be deleted by other threads, better double check.
   auto zfile = GetFile(fname);
```

Comment:
@MaisenbacherD : What is the potential race? if we hold a reference to the file, it existed at that point and because we have a reference to it will continue to be valid, so it should safe to check if it was deleted after that right?